### PR TITLE
Auto into query values

### PR DESCRIPTION
**Pull request by Jasperav**

### CODE REVIEW

**Comment by Jasperav** [comment]
Please review https://github.com/AlexPikalov/cdrs-helpers-derive/pull/7 first, because changes from that PR are also included here

File: example/Cargo.toml [hunk 1]
```diff
@@ -6,4 +6,4 @@ authors = ["Alex Pikalov <alex.pikalov.khar@gmail.com>"]
 [dependencies]
 time = "0.1.36"
 cdrs_helpers_derive = { path = ".." }
-cdrs = "2.0.0-beta.3"
+cdrs = "2.3.2"
```
Comment (Jasperav) [inline example/Cargo.toml:9]: Note: this can only be merged back when https://github.com/AlexPikalov/cdrs/pull/330 has been merged

**Comment by Jasperav** [comment]
And please also review https://github.com/AlexPikalov/cdrs/pull/330 before merging this one

**Review by Jasperav — COMMENTED** [review]
O I just wanted to leave a commit in the code but now I made myself a reviewer, this was not intended

**Review by AlexPikalov — APPROVED** [review]
Thanks @Jasperav !
And sorry for a later review.

**Pull request merged**